spec/presenters/work_provenance_display_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe WorkProvenanceDisplay do
+
+  let(:work_0) { FactoryBot.create(:work, provenance:  nil )}
+  let(:work) { FactoryBot.create(:work, provenance:  "provenance metadata" )}
+
+  it "returns an empty string if provenance is blank" do
+    expect(WorkProvenanceDisplay.new(work_0).display). to eq ""
+  end
+
+  it "correctly splits the provenance" do
+    expect {
+        WorkProvenanceDisplay.new(work_0).split_provenance
+    }.to raise_error(ArgumentError)
+
+    #no notes
+    expect(WorkProvenanceDisplay.new(work).split_provenance).
+      to contain_exactly("provenance metadata")
+
+    work.provenance = "provenance metadata \n\n NOTES: \n\n notes"
```

Comment:
I believe a space before and after `\n\n` is _atypical_ data?  Someone entering or copying or pasting isn't going to wind up with a space before or -- especially -- after.  A space after the `\n\n` would mean a space before the newline:

```
some provenance summary

NOTES:

 there's a space at the beginning of this line

but maybe not before this one
```

Am I right that's atypical data?  If it's atypical, it shouldn't be the pattern used in all the tests, most of the tests should have no space before/after `\n\n`, so we're testing typical data. If you want a test or two testing the atypical edge case  with spaces to make sure that's handled too, that's cool too, but it shouldn't be every test. 

In fact, ideally, these tests should be written with _real sample data_.  At least one/some, if not all of them, should be actual data from our sample records, not placeholder "provenance metadata".  Testing with real data instead of placeholder data makes it more likely that the test data is representative, and not dissimilar in some way that would cause a break on real data but not test data.